code/game/objects/items/reagent_containers/food/drinks/bottle.dm

```diff
@@ -115,10 +115,10 @@
 	center_of_mass = list("x"=17, "y"=3)
 	list_reagents = list("vodka" = 100)
 
-/obj/item/reagent_container/food/drinks/bottle/tequilla
-	name = "\improper Caccavo Guaranteed Quality Tequilla"
+/obj/item/reagent_container/food/drinks/bottle/tequila
+	name = "\improper Caccavo Guaranteed Quality Tequila"
 	desc = "Made from premium petroleum distillates, pure thalidomide and other fine quality ingredients!"
-	icon_state = "tequillabottle"
+	icon_state = "tequilabottle"
```

Comment:
You're changing icon states, are you sure these exist in the .dmi?

Reply:
I accordingly changed the icon_state names in the .dmi too.